components/function-controller/pkg/apis/serverless/v1alpha2/function_validation.go

```diff
@@ -221,7 +225,7 @@ func (spec *FunctionSpec) validateSources(vc *ValidationConfig) error {
 	return errors.Errorf("spec.source should contains only 1 configuration of function")
 }
 
-func validateResources(resources corev1.ResourceRequirements, minMemory, minCPU resource.Quantity, parent string) error {
+func validateResources(resources *corev1.ResourceRequirements, minMemory, minCPU resource.Quantity, parent string) error {
```

Comment:
We can remove the pointer to not have nil problem.

Reply:
Done